sdk/identity/Azure.Identity/README.md

```diff
@@ -280,5 +277,24 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 [queues_client_library]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Queues
 [eventhubs_client_library]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/eventhub/Azure.Messaging.EventHubs
 [azure_core_library]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/core/Azure.Core
-[identity_api_docs]: https://azure.github.io/azure-sdk-for-net/identity.html
+[identity_api_docs]: https://docs.microsoft.com/dotnet/api/azure.identity?view=azure-dotnet
+[vs_login_image]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/images/VsLoginDialog.png
+[vs_code_login_image]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/images/VsCodeLoginCommand.png
```

Comment:
I had to use the direct link, e.g. https://raw.githubusercontent.com/Azure/azure-sdk-for-net/master/sdk/identity/Azure.Identity/images/VsCodeLoginCommand.png, to get the images to render in Python generated docs.

Reply:
updated all the images links to use the raw endpoint